src/style.css

```
@@ -0,0 +1,32 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  display: flex;
```

```suggestion
.stars__star {
  display: flex;
```
There is no need to list all modifiers, you can set all default styles just for `stars__star` class and only change background image for modifiers